Add 11.26 release notes (#403)
mondoohq/docs: releases/2024-10-15-mondoo-11.26-is-out.md

---
slug: mondoo-11.26-is-out/
title: Mondoo 11.26 is out!
description: Announcing the 11.26 release of Mondoo with new AWS scan filtering options, custom fields in Zendesk cases, and more!
authors: [tim]
image: /img/releases/2024-10-15-mondoo-11.26-is-out/zendesk.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.26 is out! This release includes new AWS scan filtering options, custom fields in Zendesk cases, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### AWS resource scan filtering

Pick and choose the exact AWS resources that Mondoo evaluates with new scan filtering.

> The AWS Serverless integration now includes the ability to define AWS resources to scan with region, instance, and tag based include and exclude filters. Scan just a single region or exclude resources using owner tags.

Reviewer: ```suggestion
The AWS Serverless integration now includes the ability to define AWS resources to scan with region-, instance-, and tag-based include and exclude filters. Scan just a single region or exclude resources using owner tags.
```

> The AWS Serverless integration now includes the ability to define AWS resources to scan with region-, instance-, and tag-based include and exclude filters. Scan just a single region or exclude resources using owner tags.

Reviewer: ```suggestion
The AWS serverless integration now includes the ability to define AWS resources to scan with region-, instance-, and tag-based include and exclude filters. Scan just a single region or exclude resources using owner tags.
```

![AWS resource filtering](/img/releases/2024-10-15-mondoo-11.26-is-out/filtering.png)

> For CLI users, now you can specify a comma separated list of tag values with the `--filters` flag.

Reviewer: ```suggestion
For CLI users, now you can specify a comma-separated list of tag values with the `--filters` flag.
```

> To include resources with the "Owner" tag of "backend", "frontend", or "dba":

Reviewer: ```suggestion
To include only EC2 instances with the "Owner" tag of "backend", "frontend", or "dba":
```

```bash
--filters ec2:tag:Owner=backend,frontend,dba
```

> To exclude the "Owner" tag of "qa" and "dev":

Reviewer: ```suggestion
To exclude EC2 instances with the "Owner" tag of "qa" and "dev":
```

```bash
--filters exclude:ec2:tag:Owner=qa,dev
```

### Custom fields for Zendesk cases

> Improve the routing of cases generated in Zendesk with new globally set custom fields.

Reviewer: ```suggestion
Improve the routing of cases generated in Zendesk with new globally-set custom fields.
```

> Improve the routing of cases generated in Zendesk with new globally-set custom fields.

Reviewer: ```suggestion
Improve the routing of cases generated in Zendesk with new globally set custom fields.
```

![Custom fields for Zendesk](/img/releases/2024-10-15-mondoo-11.26-is-out/zendesk.png)

## 🧹 IMPROVEMENTS

### Improved policy risk display

> Understand the risk scores distribution of policies accross assets in your spaces with improved asset scan results in the security policies page.

Reviewer: ```suggestion
Understand the risk scores distribution of policies across assets in your spaces with improved asset scan results in the security policies page.
```

![Risk score distribution](/img/releases/2024-10-15-mondoo-11.26-is-out/risk_score.png)

### Ubuntu 24.10 platform support

Secure workstations running the latest release of Ubuntu with full Ubuntu 24.10 support including Linux security policies, vulnerability detection, and EOL support.

## 🐛 BUG FIXES AND UPDATES

- Fix errors on the Software page versions graph if all risk severities are unchecked.
- Add a creation date column to ticketing integration lists.
- Don't display "Last scanned" on ticketing integration pages.
- Don't display the "Schedule Now" button on ticketing integration pages.
- Don't include the "Pause Scanning" menu option on ticketing integration pages.
- Improve display of affected assets on advisories, CVEs, and checks pages.
- Fix flickering of text fields on integration setup pages.
- Improve rendering of manual setup instructions on the Workstation integration setup page.
- Don't show "Base Score" in the score summary of CVEs or Advisories.
- Fix incorrect rendering of some tool tips on overview pages.
- Improve rendering of compliance progress bars at 0% progress.
- Improve reliability of advisories and MTTR values on the vulnerabilities overview page.

> - Improve readability of markdown generated by Cases.

Reviewer: ```suggestion
- Improve readability of markdown generated by cases.
```
